src/graph/sampling/randomwalks/atomic.cuh

```diff
@@ -0,0 +1,201 @@
+/*!
```

Comment:
You don't need to duplicate the file; either `#include "../../array/cuda/atomic.cuh"` or move the `array/cuda/atomic.cuh` file into `include` directory if you don't like including with double quotes (although that will require changing some other places).